cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -718,6 +719,28 @@ public void liveInfo(HttpRequest request, HttpResponder responder, @PathParam("n
                 runtimeService);
   }
 
+  @GET
+  @Path("/apps/{app-id}/schedules/{program-type}/{program-name}")
+  public void programSchedules(HttpRequest request, HttpResponder responder,
```

Comment:
Can we call this ``getProgramSchedules`` as well? Similar change in ``AppFabricHttpHandler`` for the v2 method as well.